sessions/react-props/button/README.md

```diff
@@ -0,0 +1,79 @@
+# Button
+
+Buttons are everywhere: a green submit button, a red danger button, a disabled button because you did not fill in the form correctly, ...
+
+And here is the magic: we can only do this with one button component using props! Let's try it.
+
+## Tasks
+
+### 1. A Simple Button Component
+
+Look at the [`src/App.js`](./src/App.js) file: the `App` component returns a heading which wants to be replaced by a component.
+
+Your task is to write a `Button` component which takes three props `color`, `disabled`, and `text` and renders a button a button with the given color, text and disabled state.
+
+You can use the following hints as guideline:
+
+- Write the `Button` component inside of the [`App.js`](src/App.js).
+
+  - It accepts three props: `color`, `disabled`, and `text` (make sure to destructure them).
+  - It returns an HTML button element which
+  - renders its text according to the `text` prop,
+  - has its disabled attribute set according to the `disabled` prop, and
+  - uses the `color` prop to set the text color of the button
+    - Note: use inline styling with the help of the `style` attribute: `style={{height: "100px"}}`
+    - Additional hint: If you want to set the background color via a prop, you need to use camelCase `backgroundColor` instead of kebab-case `background-color`.
```

Comment:
```suggestion

>💡 Note: use inline styling with the help of the `style` attribute: `style={{height: "100px"}}`

>💡 Additional hint: If you want to set the background color via a prop, you need to use camelCase `backgroundColor` instead of kebab-case `background-color`.
```

Maybe this doesn't work, but I figured if these are special notes, we should format them as such. :)